pystac/eo.py

```diff
@@ -356,9 +356,9 @@ def get_bands(self):
             of asset.
         """
 
-        if not self.item:
+        if not self.owner:
```

Comment:
The Attributes Pydocs for EOItem should also be updated.

Reply:
09f883a